Description: No longer build the examples to avoid an error with OpenJDK 17
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/981854
--- a/build.xml
+++ b/build.xml
@@ -222,7 +222,7 @@
     <property name="javac.fork" value="false"/>
   </target>
 
-  <target name="build" depends="init, build.core, build.examples, build.xml,
+  <target name="build" depends="init, build.core, build.xml,
                                 build.javamail, build.jms, build.jmx"/>
 
   <target name="build.core" depends="init">
